sdk/appconfiguration/app-configuration/swagger/swagger.md

```diff
@@ -16,83 +16,83 @@ model-date-time-as-string: false
 optional-response-headers: true
 sample-generation: false
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20210111.1"
+  "@autorest/typescript": "6.0.0-beta.16"
```

Comment:
For this version, please add the option `core-http-compat-mode: true` and regenerate it please. This will ensure that the SDK is generated in compatibility mode

Reply:
done